mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodAppDelegate.m

```diff
@@ -96,6 +98,11 @@ - (id)init {
     return self;
 }
 
+-(void)applicationDidFinishLaunching:(NSNotification *)aNotification {
+    [[NSUserDefaults standardUserDefaults] registerDefaults:@{ @"NSApplicationCrashOnExceptions": @YES }];
```

Comment:
1. Is this the only place you have Keyman sending Crashlytics?
2. Is it possible to track new keyboard installations? (Either in this PR or a future one)

Reply:
Additional analytics can wait until 11.0. Crash reporting is important for managing reliability; everything else is data to help us prioritise (and we have plenty to do now anyway without needing more data!)